src/gui/syncrunfilelog.cpp

```diff
@@ -41,7 +41,7 @@ QString SyncRunFileLog::directionToStr(SyncFileItem::Direction dir)
     return re;
 }
 
-QString SyncRunFileLog::instructionToStr(csync_instructions_e inst)
+QString SyncRunFileLog::instructionToStr(SyncInsturctions inst)
```

Comment:
Note that this function only consider that it has only one value,  not used as a flag.  So it should not take the QFlags